client/components/videos-ui/index.jsx

```diff
@@ -118,26 +119,18 @@ const VideosUi = ( {
 				) }
 				<div className="videos-ui__header-content">
 					<div className="videos-ui__titles">
-						<h2>{ translate( 'Watch five videos.' ) }</h2>
-						<h2>{ translate( 'Save yourself hours.' ) }</h2>
+						<h2>{ COURSE_DETAILS[ courseSlug ].headerTitle }</h2>
```

Comment:
Non-blocking: I think this could be more readable if you extract the needed info into specific variables, but feel free to keep it this way if you don't like that. If you decide to follow my suggestion, it's fine to deploy this without asking for a re-approval.

```diff
diff --git a/client/components/videos-ui/index.jsx b/client/components/videos-ui/index.jsx
index 027006beec..7091a2ffbd 100644
--- a/client/components/videos-ui/index.jsx
+++ b/client/components/videos-ui/index.jsx
@@ -98,6 +98,8 @@ const VideosUi = ( {
 		}
 	}, [ course ] );
 
+	const { headerTitle, headerSubtitle, headerSummary } = COURSE_DETAILS[ courseSlug ];
+
 	return (
 		<div className="videos-ui">
 			<div className="videos-ui__header">
@@ -119,12 +121,12 @@ const VideosUi = ( {
 				) }
 				<div className="videos-ui__header-content">
 					<div className="videos-ui__titles">
-						<h2>{ COURSE_DETAILS[ courseSlug ].headerTitle }</h2>
-						<h2>{ COURSE_DETAILS[ courseSlug ].headerSubtitle }</h2>
+						<h2>{ headerTitle }</h2>
+						<h2>{ headerSubtitle }</h2>
 					</div>
 					<div className="videos-ui__summary">
 						<ul>
-							{ COURSE_DETAILS[ courseSlug ].headerSummary.map( ( text ) => {
+							{ headerSummary.map( ( text ) => {
 								return (
 									<li>
 										<Gridicon icon="checkmark" size={ 18 } /> { text }

```